docs/source/explore/personalize/install-apps.md

```diff
@@ -1,32 +1,32 @@
 <div class="admonition warning">
     <p class="admonition-title">Warning</p>
-    <p>This document is outdated. FIXME: mentions marketplace, mentions old inventory, mentions apps which are not widely available anymore</p>
+    <p>This document is outdated. FIXME: mentions old inventory, mentions apps which are not widely available anymore</p>
 </div>
 
 # Install Tablet Apps
 
-You can enhance your experience in Vircadia by installing tablet apps from the Marketplace. These apps enable you to get more out of Vircadia. Do you want to record your avatar dancing, take selfies, or finger paint? Or are you looking to livestream to YouTube or make your avatar clap? Our Marketplace has a number of apps that can help you customize your experience in-world.
+You can enhance your experience in Vircadia by installing tablet apps. These apps enable you to get more out of Vircadia. Do you want to record your avatar dancing, take selfies, or finger paint? Or are you looking to livestream to YouTube or make your avatar clap? There's a number of apps that can help you customize your experience in-world.
 
-If you can't find an app for what you'd like to do, you can [create your own](../../create/applications) and upload it to our Marketplace for your personal use or to sell to others.
+If you can't find an app for what you'd like to do, you can [create your own](../../create/applications) and upload it for your personal use or to sell to others.
 
 **On This Page:**
 
 * [Install Tablet Apps](#install-tablet-apps)
 * [Update an App](#update-an-app)
-* [Our Marketplace Apps](#our-marketplace-apps)
+* [Our Apps](#our-apps)
 	* [Spectator Cam: Record or Livestream in Vircadia](#spectator-cam)
 	* [Finger Painting](#finger-painting)
 	* [Text to Speech](#text-to-speech)
 
 
 ## Install Tablet Apps
-We and many users have created tablet apps that are available in Vircadia's Marketplace. To find and install an app:
+We and many users have created tablet apps that are available in Vircadia. To find and install an app:
 
-1. In Interface, pull up your HUD or Tablet and go to **Market**.
-2. Search for a specific app, or browse all apps by selecting the 'Apps, Scripts, & Tools' category.
-3. Click the app of your choice to purchase it and click the price of the item. Once you've purchased your app, you can view it in the **Inventory** app or 'Recent Activity'.
-4. After completing your purchase, click 'Install App' to start using the app in Vircadia.
-5. Once the app is installed, click 'Open App' to launch. You can also access the app through your HUD or Tablet.
+1. In Interface, pull up your HUD or Tablet and go to **More**.
+2. Search for a specific app, or browse all apps.
+3. Click the app of your choice to install it. Once you've installed your app, you can view it in the **Inventory** app or 'Recent Activity'.
```

Comment:
```suggestion
3. Click the app of your choice to install it.
```
There is no Inventory steps nowadays. Once you click "Install" in the More app, the app is installed and shows on your toolbar if it has a toolbar icon. (Some apps may not have toolbar icons as they don't have any use for them. "VR Grabscale" is one of these apps.)